packages/connectivity/src/scp-cf/token-accessor.spec.ts

```diff
@@ -240,16 +237,15 @@ describe('token accessor', () => {
       expect(retrieveFromCacheSpy).toHaveBeenCalledTimes(0);
     });
 
-    it('throws an error with only the cause property', async () => {
+    it('throws an error with the cause property', async () => {
```

Comment:
[req] Here the text was not clear. The PR from shoji was that it has the cause, but not the cause.config which contained the full axios config. Perhaps make this more clear in the test description and adjust the assertion so that it checks for that.

Reply:
The `getUserToken` method should behave the same so perhaps add a second test.